shared/lib_irradproc.cpp

```diff
@@ -1872,324 +1847,77 @@ void irrad::getBackSurfaceIrradiances(double pvBackShadeFraction, double rowToRo
 	}
 }
 
-static double vec_dot(double a[3], double b[3])
-{
-	return a[0] * b[0] + a[1] * b[1] + a[2] * b[2];
-}
-
-static void vec_cross(double a[3], double b[3], double result[3])
-{
-	result[0] = a[1] * b[2] - a[2] * b[1];
-	result[1] = a[2] * b[0] - a[0] * b[2];
-	result[2] = a[0] * b[1] - a[1] * b[0];
-}
-
-//a - b
-static void vec_diff( double a[3], double b[3], double result[3] )
-{
-	result[0] = a[0] - b[0];
-	result[1] = a[1] - b[1];
-	result[2] = a[2] - b[2];
-}
 
-static void get_vertices( double axis_tilt, double axis_azimuth, double gcr,				 
-				 double vertices[3][4][3], double rotation)
+double shadeFraction1x( double solar_azimuth, double solar_zenith,
+						double axis_tilt, double axis_azimuth,
+						double gcr, double rotation)
```

Comment:
I see I have made a mess of mixing tabs and spaces.  Should I standardize to tabs?

Reply:
Nope, they should all be spaces